### PR TITLE
feat(performance-starfish-db-widget): Minor iterations on feedback.

### PR DESCRIPTION
**Pull request by Abdkhan14**
For project: [Starfish db module widget in Performance](https://www.notion.so/sentry/Performance-Widget-for-Starfish-DB-Module-468c303b1bb44d3287f7a7c29b64f272)

Iterated on feedback, under the feature flag `performance-database-view`.

- Added loading indicator for loading state. 
- Updated query to not include '!span.op: db.redis' as it's done in the backend now.
- Replaced `Truncate` component  for span description row with `SpanDescriptionCell` for better readability of full query:
Screenshot 2023-08-31 at 4 07 36 PM

- Added alpha feature badge 
- Widgets in the top row now have `4` expandable rows instead of `3`. Adjusted chart height.
Screenshot 2023-09-04 at 3 48 35 PM

### CODE REVIEW

File: static/app/views/performance/landing/widgets/components/widgetHeader.tsx [hunk 1]
```diff
@@ -25,6 +30,7 @@ export function WidgetHeader<T extends WidgetDataConstraint>(
           ) : (
             <TextOverflow>{title}</TextOverflow>
           )}
+          {isStarfishDBWidget && <FeatureBadge type="alpha" />}
```
Comment (AbhiPrasad) [inline static/app/views/performance/landing/widgets/components/widgetHeader.tsx:33]: Could we adjust the alignment of the badge?

image

**Review by AbhiPrasad — COMMENTED** [review]
thanks for the quick turnaround!!

File: static/app/views/performance/landing/widgets/widgets/lineChartListWidget.tsx [hunk 2]
```diff
@@ -442,18 +449,20 @@ export function LineChartListWidget(props: PerformanceWidgetProps) {
...
-                    }/span/${group}?${qs.stringify({...location.query})}`}
-                  >
-                    <Truncate value={description} maxLength={40} />
-                  </GrowLink>
+                  <StyledTextOverflow>
+                    <SpanDescriptionCell
+                      projectId={projectID}
+                      group={group}
+                      description={description}
+                      moduleName={ModuleName.DB}
```
[the remark this answers is not in the document]
Comment (Abdkhan14) [reply static/app/views/performance/landing/widgets/widgets/lineChartListWidget.tsx:463]: Ahh good catch @AbhiPrasad I'll push a fix

File: static/app/views/performance/landing/widgets/components/widgetHeader.tsx (hunk 1 above)
Comment (Abdkhan14) [reply static/app/views/performance/landing/widgets/components/widgetHeader.tsx:33]: Done.

**Review by AbhiPrasad — APPROVED** [review]
🚀

**Pull request merged**